mmv1/products/edgecontainer/Cluster.yaml

```
@@ -256,6 +256,37 @@ properties:
                   An RRULE (https://tools.ietf.org/html/rfc5545#section-3.8.5.3) for how
                   this window recurs. They go on for the span of time between the start and
                   end time.
+      - !ruby/object:Api::Type::Array
+        name: "maintenance_exclusions"
```

This should match the API field name - camelCase instead of snake_case. This is probably why you're seeing a diff trying to _add_ a maintenance policy during the apply that should be _removing_ a policy.

```suggestion
        name: "maintenanceExclusions"
```

---

Noted, updated the field name to camelCase. Thanks for catching that

---

Will add new gpaste of manual test results once finished